x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -32,13 +32,22 @@ const navigationTree: NavigationTreeDefinition = {
           title: i18n.translate('xpack.serverlessObservability.nav.discover', {
             defaultMessage: 'Discover',
           }),
-          link: 'discover',
+          link: 'observability-log-explorer',
+          // prevent this entry from ever becoming active, effectively falling through to the obs-log-explorer child
+          getIsActive: () => false,
```

Comment:
Don't we want ever to mark this link as active?

Reply:
The menu hierarchy here is:

- `Logs Explorer`, which is never active and hides its breadcrumb, but needed so the Logs Explorer is the default tab
- `Discover`, which is needed so `Discover` shows up in the breadcrumbs and the entry is active when in Discover
- `Logs Explorer`, which is needed so the breadcrumbs are `Discover` / `Logs Explorer` and the entry is also active when in the Logs Explorer

It's the only way I found to get the serverless nav and breadcrumbs to behave that way. If you know of any other approach to achieve the same thing I'd gladly adopt it.

Reply:
I just checked and indeed the way multiple active node paths (here you have 2 pointing to the `observability-log-explorer` link) are declared forces you to do this. I'll open a PR to fix it in the serverless nav so you don't need to hack it here.

Reply:
That would be great. Would you prefer us to wait for your change or shall I align with it in a follow-up?

Reply:
Sorry just reading now. All good in opened my PR after you merged 👍 
https://github.com/elastic/kibana/pull/174184